wiki/about-scriptlets.md

```diff
@@ -621,6 +621,8 @@ example.org#%#//scriptlet('log', 'arg1', 'arg2')
 
 Removes content from the specified M3U file.
 
+Related UBO scriptlet:
+https://github.com/gorhill/uBlock/wiki/Resources-Library#m3u-prunejs-
```

Comment:
The wiki is automatically built during releases, so the ones you have added in the descriptions should be removed in favor of the updated scriptlets' descriptions.